app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Producer and Consumer policies
+---
+
+With namespace scoped policies in {{site.mesh_product_name}} you can have fine-grained control over policies and how they apply to 
```

Comment:
We should mention empowering app owners by taking advantage of Kubernetes RBAC

Reply:
updated